src/pages/trader-tools/pip-calculator/_pip-calculator.tsx

```diff
@@ -39,44 +38,14 @@ import {
     FormulaText,
     StyledOl,
 } from '../common/_style'
-import {
-    Accordion,
-    AccordionItem,
-    Header,
-    LocalizedLinkText,
-    QueryImage,
-    Text,
-} from 'components/elements'
+import { PipSyntheticExample, PipForexExample } from './_example-pip'
+import { Accordion, AccordionItem, Header, LocalizedLinkText, Text } from 'components/elements'
 import Input from 'components/form/input'
 import RightArrow from 'images/svg/tools/black-right-arrow.svg'
 import { Flex, Show } from 'components/containers'
 import { localize, Localize } from 'components/localization'
 
 const PipCalculator = () => {
-    const query = graphql`
-        query {
-            pip_value_formula: file(relativePath: { eq: "trade-tools/pip-value-formula.png" }) {
-                ...fadeIn
-            }
-            pip_value_forex_formula: file(
-                relativePath: { eq: "trade-tools/pip-value-forex-formula.png" }
-            ) {
-                ...fadeIn
-            }
-            pip_value_formula_mobile: file(
-                relativePath: { eq: "trade-tools/pip-value-formula-mobile.png" }
-            ) {
-                ...fadeIn
-            }
-            pip_value_forex_formula_mobile: file(
-                relativePath: { eq: "trade-tools/pip-value-forex-formula-mobile.png" }
-            ) {
-                ...fadeIn
-            }
```

Comment:
Please delete all these assets that you have removed from the images folder if they are no longer in used now. @fiona-deriv